### DISCUSSION TITLE
Icicle KIt Reference Design for VXwork7 OS.

### DISCUSSION THREAD

**Discussion by OrlyShdema** [polarfire-soc/polarfire-soc-documentation#202] [RTOS for PolarFire SoC] [answered]
Hi ,

For Icicle Kit, Do you have reference design with image that includes VxWorks 7 OS and boot up it on single core ?
Pls. advise.

Thanks,
Orly

**Comment by hughbreslin** [comment]
Hi @OrlyShdema 

This isn't something we're providing here unfortunately. You can get information on configuring and building VxWorks from Windriver's documentation and you will have to create your own HSS payload to boot your custom AMP / SMP set up.

**Comment by tim1-cpu** [comment, marked as answer]
https://labs.windriver.com/downloads/wrsdk-vxworks7-docs/660/README-polarfire-icicle.html

Regards

Tim Morin
Technical Fellow
